src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
This also makes the `StopWatchEvent::getStartTime()`, `::getEndTime()` and `getDuration()` return `int|float`, right?
So I fear it won't be enough to ensure BC :/ You'll need to be able to configure the `$morePrecision` flag higher.

Or, wouldn't it be possible to simply add a flag to the three methods mentioned above: `public function getStartTime(/**bool */$morePrecision = false)` instead?